legal-api/report-templates/correction.html

```diff
@@ -47,8 +47,10 @@
          {% if shareClasses and (shareClassesChange or newShareClasses or ceasedShareClasses) %}
             [[common/shareStructure.html]]
          {% endif %}
-
+         
          [[common/resolutionDates.html]]
+
+         [[special-resolution/resolutionApplicationCorrection.html]]
```

Comment:
So special-resolution/resolutionApplicationCorrection.html and correction/legalNameChange.html have name change related templating snippets.

Is this something we can just remove from  special-resolution/resolutionApplicationCorrection.html and use correction/legalNameChange.html?

Reply:
good catch Argus! I removed name change on csr correction and just keep name change on correction/legalNameChange.html